test.sh

```diff
@@ -61,32 +62,32 @@ then
 	fi
 
 	echo "> Creating images"
-	dd if=/dev/zero of=block count=10 bs=1M 2>&1 | sed 's/^/\t/g'
+	dd if=/dev/zero of=block count=10 bs=1M 2>&1 | sed 's/^/\t/'
```

Comment:
Actually global is required in multi-line outputs, but not for `sed`, which parses lines one by one. So this is correct.